app/api/unit_requests.rb

```diff
@@ -0,0 +1,23 @@
+require 'grape'
+
+module Api
+  class UnitRequests < Grape::API
+    helpers AuthenticationHelpers
+    helpers AuthorisationHelpers
+
+    desc 'Create unit request'  
+    params do
+           requires :unit_id, type: Integer, desc: 'Id of the unit'
+           requires :user_id, type: Integer, desc: 'Id of the unit'
+    end 
+    post '/unitrequests' do
+      unit = Unit.find(params[:unit_id])
+      stud = User.find(params[:user_id])
+
+
```

Comment:
What if they are already in the unit?
We could also ask for the role they are interested in - are they a tutor, guest, student?